scripts/superdesk-workspace/content/content.js

```diff
@@ -84,6 +84,51 @@
                 return _item;
             });
         };
+
+        /**
+         * Create new item using given content type
+         *
+         * @param {Object} contentType
+         * @return {Promise}
+         */
+        this.createItemFromContentType = function(contentType) {
+            var item = {
+                type: TEXT_TYPE,
```

Comment:
I think we will have to have some default that would be used for ingest. And should be editable imo. regarding export - I'm not sure yet. we will have to discuss this. but if it's not possible to output this with some format (like anpa) - we will just ignore it imo.